packages/node/src/handlers.ts

```diff
@@ -32,7 +32,7 @@ export function tracingHandler(): (
     // TODO: At this point `req.route.path` (which we use in `extractTransaction`) is not available
     // but `req.path` or `req.url` should do the job as well. We could unify this here.
     const reqMethod = (req.method || '').toUpperCase();
-    const reqUrl = req.url;
+    const reqUrl = req.url && stripUrlPath(req.url);
```

Comment:
That would mean adding `undefined` to both the input and output types, which then would just push the `&&`ing to the other end (because it the result wouldn't be guaranteed to be defined) - no? Is that better? (Happy to do it if you think so.)